script/Arcueid/Day 10/10_00_ARC10_5E.txt

```diff
@@ -359,19 +357,19 @@ Arcueid crouches low to the ground, panting heavily, like a hulking predator get
 [sha:f7514b9e2d2658b91100f1d773e028b16015b3c3]{
 -- Page 649, Offset 26002.
 -- 「あーあ、ひでえもんだ。そこいらのグールと大差ねえじゃねえか。それでも最後の真祖かよ、姫様。ラウレンティスのジジイが見たらさぞ失望するだろうぜ」
-"Aaah, what an awful sight. You're no different from a Ghoul.
- And yet you're supposed to be the last of the True Ancestors, princess?
- Old man Laurentis would be disappointed if he saw this."
+"Boo-fucking-hoo, what a godawful sight. Ain't much different from some trashy-ass Ghoul.
+ Am I really supposed to believe you're the last True Ancestor, Princess?
+ Man, if Ol' Laurentis saw you right now, he'd be real bummed 'bout ya."
```

Comment:
Last phrase here doesn't really parse well to me. Bummed just feels too fun/light.

Reply:
I agree, in hindsight. Maybe the first one is a bit too much as well.